New package: WebP v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: WebP
- Repository: https://github.com/stemann/WebP.jl
- Created by: @stemann
- Version: v0.1.0
- Commit: 0a2e0dbaa27ee52e868e56d7524014b948bc7e5a
- Git reference: HEAD

LilithHafner commented:
[noblock]

Name LGTM, but will require manual merge because it's too short for automerge to accept.

stemann commented:
> I'll also note that it is going to be fairly hard for other folks to make use of your package right now because it doesn't have documentation. A brief usage example in the readme & a link to the upstream docs & instructions for how to interpret the upstream docs for using this package would go a long way; and full docs for the wrapper would make it even better!
> 
> Thanks for the package!

You're all very welcome :-)

Good point regarding documentation. I have elaborated some on the README - with usage examples - the intent is to provide an idiomatic Julia API (not expose and document the wrapped C API).

[noblock]

JuliaHubRegistrator committed:
New package: WebP v0.1.0

UUID: e3aaa7dc-3e4b-44e0-be63-ffb868ccd7c1
Repo: https://github.com/stemann/WebP.jl.git
Tree: 6202da13a82d60a0574602eee7f3b332ed17451c

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09
